.github/workflows/audit.yml

```diff
@@ -23,5 +23,5 @@ jobs:
       - name: Update npm to latest
         run: npm i --prefer-online --no-fund --no-audit -g npm@latest
       - run: npm -v
-      - run: npm i --ignore-scripts --no-audit --no-fund --package-lock
+      - run: npm i --ignore-scripts --no-audit --no-fund
```

Comment:
the `--package-lock` was here on purpose because the next step runs `npm audit` and that needs a package-lock to exist

Reply:
Good call! This was a result of moving it to a shared partial. I'll make it configurable.